spec.bs

```diff
@@ -70,11 +81,129 @@ dictionary AuctionAdInterestGroup {
 };
 </xmp>
 
+<div algorithm="joinAdInterestGroup()">
+
+The <dfn for=Navigator method>joinAdInterestGroup(|group|, |durationSeconds|)</dfn> method steps are:
+
+1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the
+  "join-ad-interest-group" [=policy-controlled feature=], then return [=a promise rejected with=] a

```

Comment:
I sent a pull request (https://github.com/w3c/webappsec-permissions-policy/pull/438) for this last Nov/Dec, and it's still open. I'll follow up on that PR to get it merged.

Reply:
On [that PR](https://github.com/w3c/webappsec-permissions-policy/pull/438#issuecomment-1000454791), you said you'd have this PR add the default allowlist definition Mike asked for. You should probably add it here, and possibly merge this PR, before that PR gets merged.

Reply:
Mike asked for the default allowlist definition in FLEDGE explainer, which has been [added](https://github.com/WICG/turtledove/pull/249/files).
Currently the links I provided in that[ permissions policy PR 438](https://github.com/w3c/webappsec-permissions-policy/pull/438) for the two features were links to the FLEDGE explainer. After we have the spec, I think we can change the links to the spec (maybe wait until the spec has both joinAdInterestGroup and runAdAuction in place).
But since it sounds merging this PR doesn't require [PR 438](https://github.com/w3c/webappsec-permissions-policy/pull/438) gets merged, I'll request merging PR 438 later.